federation/apis/federation/deep_copy_generated.go

```diff
@@ -23,108 +23,133 @@ package federation
 import (
 	api "k8s.io/kubernetes/pkg/api"
 	conversion "k8s.io/kubernetes/pkg/conversion"
+	reflect "reflect"
 )
 
 func init() {
 	if err := api.Scheme.AddGeneratedDeepCopyFuncs(
-		DeepCopy_federation_Cluster,
-		DeepCopy_federation_ClusterCondition,
-		DeepCopy_federation_ClusterList,
-		DeepCopy_federation_ClusterSpec,
-		DeepCopy_federation_ClusterStatus,
-		DeepCopy_federation_ServerAddressByClientCIDR,
+		conversion.GeneratedDeepCopyFunc{Fn: DeepCopy_federation_Cluster, InType: reflect.TypeOf(func() *Cluster { var x *Cluster; return x }())},
```

Comment:
If I remember right, this was just an artifact of non-uniform pointer/non-pointer signatures of the deepcopy functions. Now the `in` parameter is always a pointer. Here is a simplification PR: https://github.com/kubernetes/kubernetes/pull/30341